dataland-backend/src/main/kotlin/org/dataland/datalandbackend/frameworks/gdv/model/allgemein/sonstige/GdvAllgemeinSonstige.kt

```diff
@@ -0,0 +1,13 @@
+// <--WARNING--> THIS FILE IS AUTO-GENERATED BY THE FRAMEWORK-TOOLBOX AND WILL BE OVERWRITTEN
+package org.dataland.datalandbackend.frameworks.gdv.model.allgemein.sonstige
+
+import org.dataland.datalandbackend.model.enums.commons.YesNo
+
+/**
+ * The data-model for the Sonstige section
+ */
+data class GdvAllgemeinSonstige(
+    val ausrichtungAufDieUnSdgsUndAktivesVerfolgen: String?,
```

Comment:
Is this name correct?

Reply:
ja, "UN" steht für united nations (wahrscheinlich)

Reply:
UN SDGs sind die sustainable development goals der UN.  Also passt der Name aus meiner Sicht.

Reply:
Ok, then i will close it. But man, some of the field names are wild